baker/algolia/deleteAlgoliaIndex.tsx

```diff
@@ -0,0 +1,38 @@
+import { ALGOLIA_INDEXING } from "../../settings/serverSettings.js"
+import { getAlgoliaClient } from "./configureAlgolia.js"
+import { SearchIndexName } from "../../site/search/searchTypes.js"
+import { getIndexName } from "../../site/search/searchClient.js"
+
+const deleteAlgoliaIndex = async () => {
+    if (!ALGOLIA_INDEXING) return
```

Comment:
Recall that most staging servers do not get their own Algolia index, only ones with `-algolia` in the branch name, which ends up setting `ALGOLIA_INDEX_PREFIX`.

I would skip or abort if `ALGOLIA_INDEX_PREFIX` is empty, so that we don't accidentally have code that deletes the prod index.

Reply:
Good idea, that feels much safer.